docs/index.md

```diff
@@ -155,7 +155,7 @@ Mocha allows you to use any assertion library you wish.  In the above example, w
 
 ## Asynchronous Code
 
-Testing asynchronous code with Mocha could not be simpler! Simply invoke the callback when your test is complete. By adding a callback (usually named `done`) to `it()`, Mocha will know that it should wait for this function to be called to complete the test.
+Testing asynchronous code with Mocha could not be simpler! Simply invoke the callback when your test is complete. By adding a callback (usually named `done`) to `it()`, Mocha will know that it should wait for this function to be called to complete the test. This callback accepts both an `Error` instance, or `undefined`.
```

Comment:
#3134 isn't quite accurate about what needs to be documented.

We need to document that *any non-falsy, non-`Error` instance* passed into `done()` will cause Mocha to fail.

Subclasses of `Error` are allowed.

Reply:
Can you check if that is accurate?